### PR TITLE
Buldakov lab #1

### PR DESCRIPTION
**Pull request by Wropwtr**

### CODE REVIEW

File: modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    // Arrange
+    ComplexNumber x(1.1, -1.4);
+    ComplexNumber y(1.4, -1.1);
+
+    // Act
+    ComplexNumber resut = x * y;
+
+    // Assert
+    EXPECT_EQ(-0.0, resut.getRe());
+    EXPECT_EQ(-3.17, resut.getIm());
```
Comment (kiselik) [inline modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp]: I read,that floating point operations can lead to rounding errors. For example, you can get next situation: EXPECT_EQ (-3.17, -3.171). The result of this test is false. That's why you need to use special asserts for this numbers: EXPECT_DOUBLE_EQ or EXPECT_FLOAT_EQ

Comment (Wropwtr) [inline modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp]: Thanks, i replaced unstable asserts

**Comment by Wropwtr** [comment]
Rewiew my code, please
@kiselik
@Lyzzerg
@Vagik

**Comment by Lyzzerg** [comment]
:+1:

File: modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Buldakov Vladislav
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Buldakov_Vladislav_ComplexNumberTest, Can_Create_Default) {
+    // Arrange & Act
+    ComplexNumber z = ComplexNumber();
```
Comment (Lyzzerg) [inline modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp]: А имеет ли смысл писать ComplexNumber z = ComplexNumber(); если при ComplexNumber z; всё равно вызовется конструктор по умолчанию?

**Comment by kiselik** [comment]
👍

**Comment by Lyzzerg** [comment]
👍

File: modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp (hunk 2 above)
Comment (Wropwtr) [inline modules/complex-number/test/test_buldakov_vladislav_complex_number.cpp]: Действительно, не имеет.

**Pull request merged**